.github/workflows/docker-image.yml

```
@@ -8,17 +8,21 @@ on:
 
 jobs:
   build:
+    name: Build Docker Image
     runs-on: ubuntu-latest
-
     steps:
       - uses: actions/checkout@v3
-      - name: Build the Docker image
-        run: docker build --build-arg STICKERIFY_TOKEN=${{ secrets.STICKERIFY_TOKEN }} -t rob93c/stickerify:latest .
-
-      - uses: actions/checkout@v3
-      - name: Perform Docker login
-        run: echo ${{ secrets.DOCKER_PASSWORD }} | docker login -u ${{ secrets.DOCKER_USERNAME }} --password-stdin
-
-      - uses: actions/checkout@v3
-      - name: Upload the latest Docker image
-        run: docker push rob93c/stickerify:latest
+      - uses: docker/setup-buildx-action@v2
+      - uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+      - uses: docker/build-push-action@v3
+        with:
+          build-args: |
```

Maybe it can work without creating a multiline value
```suggestion
          build-args: STICKERIFY_TOKEN=${{ secrets.STICKERIFY_TOKEN }}
```